src/test/java/teammates/test/cases/logic/CoursesLogicTest.java

```diff
@@ -28,6 +28,7 @@
 import teammates.storage.api.CoursesDb;
 import teammates.storage.api.InstructorsDb;
 import teammates.test.driver.AssertHelper;
+import teammates.test.driver.StringHelperExtension;
```

Comment:
I think this will cause the build to fail

Reply:
My bad, it comes when i copied the code from `CourseAttributesTest` but I forgot to remove.